### PR TITLE
Use ccache on Travis

### PR DESCRIPTION
**Pull request by fabiencastan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,7 +36,11 @@ matrix:
 # This results in a 2×2x2 build matrix.
 # Where the variables are: os / compiler / python version
 
+language: cpp
```
Comment (cchampet) [inline .travis.yml]: Duplicate with line 1

File: .travis.yml [hunk 2]
```diff
@@ -124,6 +128,11 @@ addons:
     build_command: "tools/travis/build.sh"
     branch_pattern: coverity_scan
 
+
+# Before uploading the new cache archive
```
Comment (cchampet) [inline .travis.yml:130]: I think 'before_cach' is self-sufficient.

**Comment by cchampet** [comment]
I'm looking forward to see the build time of Travis now! :+1:

**Comment by fabiencastan** [comment]
3min50 when there is no code change.

**Pull request merged**